docs/tutorials/services/navigate-with-rover-base.md

```diff
@@ -0,0 +1,644 @@
+---
+title: "Navigate with a Rover Base"
+linkTitle: "Navigate with a Rover"
+type: "docs"
+description: "Introduction to using a rover base with the navigation service."
+image: "/tutorials/navigate-with-rover-base/leo-in-office.png"
+imageAlt: "The leo rover navigating in the Viam Lab."
+images: ["/tutorials/navigate-with-rover-base/leo-in-office.png"]
+tags:
+  [
+    "try viam",
+    "navigation",
+    "movement sensor",
+    "vision",
+    "motion",
+    "movement sensor",
+    "base",
+    "rover",
+    "motion planning",
+    "obstacles",
+    "waypoints",
+  ]
+authors: ["Sierra Guequierre"]
+languages: ["python", "go"]
+viamresources: ["navigation", "base", "movement sensor"]
```

Comment:
this needs to use the names as in the configuration - otherwise the selection UI won't work
```suggestion
viamresources: ["navigation", "base", "movement_sensor"]
```

Reply:
Thank you, done!